Class-Code/Elementary-CA/awu/sketch.js

```diff
@@ -1,14 +1,119 @@
-//Your Name(s)
-//Project Name
-//Date
+//Your Name(s) Alex Wu
+//Project Name Elementary CA peoject
+//Date FEB 25
 
-/* A brief description of what the program is */
+/* A brief description of what the program is 
+to mimic the CA operation by javascript*/
+const rules = [0, 0, 0, 1, 1, 0, 1, 1]
+let ca = []
+let cell_size
```

Comment:
Generally, in JS the convention is to use [camelCase](https://www.w3schools.com/js/js_conventions.asp) though it's not strictly necessary, so instead of `cell_size` you would use `cellSize`